spyder/plugins/ipythonconsole/tests/test_ipythonconsole.py

```diff
@@ -1151,17 +1150,15 @@ def test_restart_kernel(ipyconsole, mocker, qtbot):
 
     # Check that we try to show Matplotlib backend errors at the beginning and
     # after the restart.
-    assert ClientWidget._show_mpl_backend_errors.call_count == 2
+    assert ClientWidget._show_mpl_backend_errors.call_count == 1
```

Comment:
Why did you changed this? This is what the test is test is precisely checking, as you can read in the comment above.

Reply:
Because the test now wait for the console to be up, the first error is shown before the test starts, and is therefore not caught. I will change that to catch it again

Reply:
Fixed